config/locales/client.en.yml

```
@@ -4191,6 +4191,12 @@ en:
               draft_count:
                 one: "%{count} draft"
                 other: "%{count} drafts"
+            review:
+              content: "Review"
+              title: "review"
+              pending_count:
+                one: "%{count} pending"
+                other: "%{count} pendings"
```

The word "pending" doesn't have a plural form. I can't find a reference that says so explicitly, but in all usage that I have seen, it is always singular, we should drop the s here.
```suggestion
                other: "%{count} pending"
```

---

+1